stellarphot/settings/custom_widgets.py

```diff
@@ -1075,3 +1077,141 @@ def _confirmation(self, change=None):
             self.fitsopen.file_chooser.reset()
             self.info_box.value = ""
             self.run_settings = None
+
+
+class TessPhotometrySetup(ipw.VBox):
+    """
+    Widget for getting some photometry inputs for a TESS Object of Interest (TOI).
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Define the children of self
+        self.header = ipw.HTML("<h2>Enter the TIC ID of the exoplanet candidate</h2>")
+        self.drop_label = ipw.HTML("How would you like to specify the TIC ID?")
+        self.drop = ipw.Dropdown(
+            options=(("Type in TIC ID", 0), ("Choose an image", 1))
+        )
+        self.tic_id_entry = ipw.IntText(description="TIC ID")
+        self.fits_opener = FitsOpener()
+        self.confirm = Confirm()
+        self.spinner = Spinner(message="<h3>Downloading TIC info...</h3>")
+        self.all_done = ipw.HTML(
+            "<h2>Files have been written, you can close this notebook.</h2>"
```

Comment:
*Suggestion, not demand.* I would remove "you can close this notebook" since I could envision a notebook with this TESS selection at the top and other code below.  Seemed like "hard coding" the solution.  You could instead provide links to the files created to be viewed before proceeding.

Reply:
Fixed that, thanks.